packages/c/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ryanmjacobs/c
+TERMUX_PKG_DESCRIPTION="Compile and execute C "scripts" in one go!"
+TERMUX_PKG_LICENSE=MIT
+TERMUX_PKG_MAINTAINER=@termux
+TERMUX_PKG_VERSION=0.14
+TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=2b66d79d0d5c60b8e6760dac734b8ec9a7d6a5e57f033b97086821b1985a870b
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_DEPENDS=bash
+TERMUX_PKG_PLATFORM_INDEPENDENT=true
+
+termux_step_make_install() {
+  local bin="$(basename $TERMUX_PKG_HOMEPAGE)"
+  install -D "$bin" -t "$TERMUX_PREFIX/bin"
```

Comment:
Use tabs for indentation in Shell scripts, as per <a href="https://github.com/termux/termux-packages/wiki/Coding-guideline">Coding guideline</a>.